packages/wdio-browserstack-service/src/launcher.js

```diff
@@ -21,7 +21,7 @@ export default class BrowserstackLauncherService {
             key: this.config.key,
             forcelocal: true,
             onlyAutomate: true,
-            ...this.options.browserstackOpts
+            ...this.options.opts
```

Comment:
Isn't this a breaking change?

Reply:
And shouldn't we also have a UT for the `opts`?

Reply:
yes, it is a breaking change and therefor will only land in v6

Reply:
Tnx, will you also add this to the breaking change changelog?

Reply:
made a remark here: https://github.com/webdriverio/webdriverio/pull/5101/files#diff-90d986c54b09e541792ff82a6913a4bfR133 .. also the PR will be marked as breaking change and will land like this in the changelog